tests/Doctrine/Tests/Annotations/PerformanceTest.php

```diff
@@ -22,7 +20,7 @@ class PerformanceTest extends TestCase
      */
     public function testCachedReadPerformanceWithInMemory()
```

Comment:
Does it still test cached reading?

Reply:
Hehe, no, needs to be dropped, redundant with the next bench.